stable/nginx-ingress/values.yaml

```diff
@@ -5,7 +5,7 @@ controller:
   name: controller
   image:
     repository: quay.io/kubernetes-ingress-controller/nginx-ingress-controller
-    tag: "0.15.0"
+    tag: "0.17.1"
```

Comment:
Can you update the readme to match the values.yaml?

Reply:
Done.